pygerrit2/rest/__init__.py

```diff
@@ -307,6 +308,14 @@ def set_message(self, message):
         """
         self.message = message
 
+    def set_tag(self, tag):
```

Comment:
github won't let me add a comment down there, but please also add handling of `self.tag` in the `__str__` method (L369).

Reply:
Done